subsystems/drivebase/SwerveDrive.java

```diff
@@ -311,8 +331,35 @@ public void mustangPeriodic() {
         Logger.recordOutput(DRIVEBASE_PITCH, getPitch());
         Logger.recordOutput(DRIVEBASE_ROLL, getRoll());
 
+        if (kConfig.kDriveMotorType.equals(Motor_Type.KRAKEN_X60)) {
+            double latestDriveCurrentDraw = 0;
+            for (int i = 0; i < mModules.length; i++) {
+                latestDriveCurrentDraw += ((TalonFX) mModules[i].getDriveMotor()).getSupplyCurrent().getValueAsDouble();
+            }
+
+            index = (index + 1) % RobotConstantsBase.SwerveDriveBase.kCurrentSampleSize; // 0-249
```

Comment:
I think this is pretty much correct except that you're going to want to update the index and bufferFull after doing the averageCurrent calculation